.github/workflows/ci.yml

```
@@ -61,7 +62,7 @@ jobs:
         working-directory: packages/fuels
 
       - name: Build Sway test projects
-        run: forc build --terse --error-on-warnings
+        run: forc build --terse
```

revert this when `sway` fixes warnings in the `std`(this will probably happen before merging this PR)
```suggestion
        run: forc build --terse --error-on-warnings
```